app/deliver_grant_funding/routes.py

```diff
@@ -67,7 +67,7 @@ def grant_setup_ggis() -> ResponseReturnValue:
     )
 
 
-@deliver_grant_funding_blueprint.route("/grants/setup/name", methods=["GET", "POST"])
+@deliver_grant_funding_blueprint.route("/grant/setup/name", methods=["GET", "POST"])
```

Comment:
These maybe could reflect that they're not consistent pages with the `/grant` route sand can represent that they're separate with `/grant-setup/name` etc.

I don't think this is important but they don't feel the same from a glance